french/script/story/st_00.rpy

```diff
@@ -6,901 +6,901 @@ translate french s00_last_time_on_jpde_265b952a:
 
     # adt sun neutral "Betrayal, then."
-    adt sun neutral ""
+    adt sun neutral "Une trahison, donc."
 
 # game/script/story/st_00.rpy:163
 translate french s00_birth_of_the_red_fang_02ee7053:
 
     # sk sun neutral "I have no wish to kill you and turn you into a martyr, Adam...{w=0.5} But you've had more than enough warnings and I still lead the Fang.{w=0.3} Popularity or not."
-    sk sun neutral ""
+    sk sun neutral "Je ne souhaite aucunement te tuer et te transfirler en un martyr, Adam...{w=0.5} Mais tu as été averti bien plus de fois que nécessaire et je reste maîtresse du Fang.{w=0.3} Populaire ou non."
 
 # game/script/story/st_00.rpy:167
 translate french s00_birth_of_the_red_fang_e9d189c3:
 
     # sk nulla "There's no betrayal in reeling in the chain of a rabid dog."
-    sk nulla ""
+    sk nulla "Il n'y a aucune trahison à secouer la chaîne d'un chien enragé."
 
 # game/script/story/st_00.rpy:168
 translate french s00_birth_of_the_red_fang_1a45b35c:
 
     # adt sun neutral "I agree.{w=0.3} A necessary sacrifice, for the good of the cause."
-    adt sun neutral ""
+    adt sun neutral "Je l'admets.{w=0.3} Un sacrifice nécessaire, pour le bien de tous."
 
 # game/script/story/st_00.rpy:169
 translate french s00_birth_of_the_red_fang_909abb05:
 
     # adt sun amused "But I wasn't making an observation.{w=0.3} I was giving {i}an order{/i}."
-    adt sun amused ""
+    adt sun amused "Mais je ne faisais aucune observation.{w=0.3} Je donnais {i}un ordre{/i}."
 
 # game/script/story/st_00.rpy:182
 translate french s00_birth_of_the_red_fang_8191c4ed:
 
     # sk nulla "Berry!{w=0.3} What are you doing?!"
-    sk nulla ""
+    sk nulla "Berry !{w=0.3} Qu'est-ce qu'il te prend ?!"
 
 # game/script/story/st_00.rpy:183
 translate french s00_birth_of_the_red_fang_86d7dcc1:
 
     # adt sun amused "Seems like you've underestimated my 'popularity', Sienna…{w=0.3} Do you see, now?"
-    adt sun amused ""
+    adt sun amused "Il semblerait que tu aies sous-estimé ma \"popularité\", Sienna...{w=0.3} La vois-tu, maintenant ?"
 
 # game/script/story/st_00.rpy:185
 translate french s00_birth_of_the_red_fang_ea858cd6:
 
     # adt sun neutral "You're right.{w=0.3} Vision alone won't suffice in shaping our future.{w=0.3} Nor would it have brought even your most trusted aides to my side."
-    adt sun neutral ""
+    adt sun neutral "Tu as raison.{w=0.3} Une vision seule ne suffira pas à bâtir notre futur.{w=0.3} Cela n’aurait pas non plus amené à mes côtés même tes collaborateurs les plus fiables."
 
 # game/script/story/st_00.rpy:186
 translate french s00_birth_of_the_red_fang_b9a72e6e:
 
     # adt sun frown "What matters is {i}results{/i}, Sienna.{w=0.3} You call me rabid, underestimate me.{w=0.3} You can't see that my passion is but a fuel."
-    adt sun frown ""
+    adt sun frown "Ce qui importe sont {i}les résultats{/i}, Sienna.{w=0.3} Tu me qualifie d'enragé et me sous-estime.{w=0.3} Tu ne peux pas voir ma passion mais un combustible."
```

Comment:
"Tu me qualifie d'enragé et me sous-estime." -> "Tu me qualifies d'enragé et me sous-estimes."

tu peux remplacer le "et" par une virgule.

"Tu ne peux pas voir ma passion mais un combustible." erreur de trad 
"[...] my passion is but a fuel." -> "[...] ma passion n'est qu'un carburant."

"Tu ne peux voir que ma passion n'est qu'un carburant."

Reply:
"Tu ne peux voir ma passion que comme un carburant."

Tu en penses quoi ?